src/textual/reactive.py

```diff
@@ -278,6 +400,8 @@ def __set__(self, obj: Reactable, value: ReactiveType) -> None:
         public_validate_function = getattr(obj, f"validate_{name}", None)
         if callable(public_validate_function):
             value = public_validate_function(value)
+        if self._validate_method:
+            value = self._validate_method.__get__(self)(value)
```

Comment:
This feels like it's ok to have 2+ validate methods but the validate decorator can only be used once per reactive. Am I reading this correctly and is this the intended behaviour?

Reply:
You can't have two validate methods via the naming convention, because they will clash. i.e. If you have two `validate_foo` methods, only the second one will be called.

You *can* have a validate method defined via the naming convention, and one via the decorators. 

I could prohibit that, but then it may break existing code.

It is a bit of a wart. I can see us removing the naming convention before 1.0.

Reply:
Right.